docs/lazy-adr/adr-002-mempool.md

```diff
@@ -15,7 +15,7 @@ For now, mempool implementation from lazyledger-core/Tendermint will be used.
 
 * inherit all limitations of the tendermint mempool
   * no prioritization of Txs
-  * many [open issues](https://github.com/tendermint/tendermint/issues?q=is%3Aissue+is%3Aopen+mempool+label%3AC%3Amempool)
+  * many [open issues](https://github.com/cometbft/cometbft/issues?q=is%3Aissue+is%3Aopen+mempool+label%3AC%3Amempool)
```

Comment:
I would leave this as is (without changing to cometbft).

If we're willing to re-import latest & greatest mempool from combetbft, we should do this in separate PR.